utils/evm/wallets.ts

```diff
@@ -50,3 +52,121 @@ export const CHAIN_BY_ID = DEFAULT_SUPPORTED_CHAINS.reduce(
   },
   {} as Record<number, (typeof DEFAULT_SUPPORTED_CHAINS)[number]>
 );
+const _SUPPORTED_WALLETS = {
```

Comment:
TIL `Object.keys` preserves string keys in the order in which they were added to the object.

```js
const obj = { b: 'banana', a: 'apple', c: 'cherry' };
console.log(Object.keys(obj)); // ["b", "a", "c"]
//VM375:2 (3) ['b', 'a', 'c']
```

Reply:
is that comment linked to that specific line?

Reply:
It was directed at your comment about keeping the one entry first. I used to think that wouldn’t work but learned it does-thanks!